### PR TITLE
Feature/templates

### PR DESCRIPTION
**Pull request by CSergienko**
@Pringels Please have a look here, let me know what you think.

### CODE REVIEW

File: quizzical/app/components/Question.js [hunk 1]
```diff
@@ -11,11 +11,39 @@ class Question extends Component {
...
+                          <label className="Question-label" for="question3">
+                              <span className="Question-checkbox"></span>
+                              3.14159265359
+                          </label>
+                      </li>
+                  </ul>
+              </fieldset>
+              <button type="submit" className="Questions-submit">
+                  Next Question
+              </button>
```
Comment (pringelmann) [inline quizzical/app/components/Question.js:43]: We will need to make this a `<Link>` component - but I will take care of that

File: quizzical/app/components/QuestionList.js [hunk 2]
```diff
@@ -23,16 +23,19 @@ class QuestionList extends Component {
...
-                        })
-                    }
-                 </ul>
-            </div>
+            <section class="Questions">
+                {
+                    this.props.questions.map((question) => {
+                        return (
+                            <div class="Questions-item">
+                                <form action="">
```
Comment (pringelmann) [inline quizzical/app/components/QuestionList.js:31]: Why is this a form?

File: quizzical/app/components/Question.js (hunk 1 above)
Comment (CSergienko) [inline quizzical/app/components/Question.js:43]: Let's discuss the details. I'd like to help on that front.

File: quizzical/templates/quizzical/index.html [hunk 3]
```diff
@@ -7,20 +7,15 @@
...
-                {% endfor %}
-            </ul>
-        {% else %}
-            <p>No questions available.</p>
-        {% endif %} -->
 
-        <div id="quizzical-app"></div>
+        <header class="Header Header--floating">
+            <a href="#" class="Header-brand">Quizzical</a>
+        </header>
```
Comment (pringelmann) [inline quizzical/templates/quizzical/index.html:17]: We can do this using components in App.js

File: quizzical/app/components/QuestionList.js (hunk 2 above)
Comment (CSergienko) [inline quizzical/app/components/QuestionList.js:31]: All the questions sit in a form - that's the idea in any case. Is there a better way you can think of doing it?

**Comment by pringelmann** [comment]
+1

**Pull request merged**